code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -522,18 +522,20 @@
 
 /mob/living/simple_animal/pet/dog/corgi/narsie/Life()
 	..()
+	if(health <= 0)
+		return
 	for(var/mob/living/simple_animal/pet/P in range(1, src))
-		if(P != src && !istype(P,/mob/living/simple_animal/pet/dog/corgi/narsie))
-			visible_message("<span class='warning'>[src] devours [P]!</span>", \
-			"<span class='cult big bold'>DELICIOUS SOULS</span>")
-			playsound(src, 'sound/magic/demon_attack1.ogg', 75, TRUE)
-			narsie_act()
-			if(P.mind)
-				if(P.mind.hasSoul)
-					P.mind.hasSoul = FALSE //Nars-Ian ate your soul; you don't have one anymore
-				else
-					visible_message("<span class='cult big bold'>... Aw, someone beat me to this one.</span>")
-			P.gib()
+			if(P != src && !istype(P,/mob/living/simple_animal/pet/dog/corgi/narsie))
+				visible_message("<span class='warning'>[src] devours [P]!</span>", \
+				"<span class='cult big bold'>DELICIOUS SOULS</span>")
+				playsound(src, 'sound/magic/demon_attack1.ogg', 75, TRUE)
+				narsie_act()
+				if(P.mind)
+					if(P.mind.hasSoul)
+						P.mind.hasSoul = FALSE //Nars-Ian ate your soul; you don't have one anymore
+					else
+						visible_message("<span class='cult big bold'>... Aw, someone beat me to this one.</span>")
+				P.gib()
```

Comment:
I think you've accidentally left these lines indented, with the new return these lines shouldn't be changed at all

Reply:
Yeah web edit gang